pages/advanced/performance.mdx

```diff
@@ -0,0 +1,111 @@
+# Performance
+
+As a library that serves the very fundamental functionality of all kinds of web apps, one of SWR’s top priorities is **performance**. 
+
+Although most of the time the actual request won’t be triggered because of SWR’s built-in **caching** and **[deduplication](/advanced/performance#deduplication)**, 
+the performance of the `useSWR` hook itself still matters. In a complex app, there could be hundreds of `useSWR` calls in a single page render.
+
+SWR uses the following techniques to ensure there is:
+- _no unnecessary requests_
+- _no unnecessary re-renders_
+- _no unnecessary code imported_
+
+without any code changes from you.
+
+## Deduplication
+
+It’s very common to have SWR hooks reused in your app. For example you have an app to render the current user’s avatar 5 times:
+
+```jsx
+function useUser () {
+  return useSWR('/api/user', fetcher)
+}
+
+function Avatar () {
+  const { data, error } = useUser()
+
+  if (error) return <Error />
+  if (!data) return <Spinner />
+
+  return <img src={data.avatar_url} />
+}
+
+function App () {
+  return <>
+    <Avatar />
+    <Avatar />
+    <Avatar />
+    <Avatar />
+    <Avatar />
+  </>
+}
+```
+
+Each `<Avatar>` component has a `useSWR` hook inside. Since they have the same SWR key and are rendered at the almost same time, **only 1 request will be made**.
```

Comment:
```suggestion
Each `<Avatar>` component has a `useSWR` hook inside. Since they have the same SWR key and are rendered at the almost same time, **only 1 network request will be made**.
```

Think it's good to explicitly say "network request" instead of just "request" (which could be image request, etc..)